src/api/api.ts

```diff
@@ -12,6 +13,11 @@ export function serveApi(
     res.send(store.getState())
   })
 
+  app.post('/registerWarmup', (req, res) => {
```

Comment:
This is basically the CMD Loader component I am mentioning.

The question is how fat do we want our drivers vs CMD Loader and vice versa. Because they are closely connected.

They are both kind of drivers and interact with the outside world. We'll think some more and try to fit it as we progress.